inst/extdata/PA2022CH_fr_exec_summary/real-estate-chapter.Rmd

```diff
@@ -3,7 +3,7 @@
 
 La section Immobilier et Hypothèques permet aux institutions financières de mesurer l'alignement de leur portefeuille immobilier et hypothécaire suisse avec l'objectif climatique du secteur du bâtiment suisse d'atteindre des émissions nettes nulles en 2050. Le test PACTA évalue donc les émissions de CO\textsubscript{2} de vos investissements immobiliers et hypothécaires.
 
-Les émissions du parc immobilier suisse représentent actuellement un peu plus d'un quart des émissions totales de CO\textsubscript{2} de la Suisse. Les nouveaux bâtiments sont aujourd'hui largement équipés de systèmes de chauffage renouvelables. D'ici 2020, les émissions de CO\textsubscript{2} de l'ensemble des bâtiments devraient avoir diminué d'au moins 40\% par rapport à 1990 - cet objectif n'a pas été atteint. Pour atteindre l'objectif "net zéro" déclaré par le Conseil fédéral en 2050, le parc immobilier suisse doit réduire plus rapidement ses émissions de CO\textsubscript{2} à l'avenir, notamment en rénovant les bâtiments existants. (\href{lien}{\url{https://www.bafu.admin.ch/bafu/fr/home/themes/climat/info-specialistes/reduction-emissions/objectifs-reduction/objectif-2050/strategie-climatique-2050.html}})
+Les émissions du parc immobilier suisse représentent actuellement un peu plus d'un quart des émissions totales de CO\textsubscript{2} de la Suisse. Les nouveaux bâtiments sont aujourd'hui largement équipés de systèmes de chauffage renouvelables. D'ici 2020, les émissions de CO\textsubscript{2} de l'ensemble des bâtiments devraient avoir diminué d'au moins 40\% par rapport à 1990 - cet objectif n'a pas été atteint. Pour atteindre l'objectif "net zéro" déclaré par le Conseil fédéral en 2050, le parc immobilier suisse doit réduire plus rapidement ses émissions de CO\textsubscript{2} à l'avenir, notamment en rénovant les bâtiments existants \textbf{\href{https://www.bafu.admin.ch/bafu/fr/home/themes/climat/info-specialistes/reduction-emissions/objectifs-reduction/objectif-2050/strategie-climatique-2050.html}{(lien)}}.
```

Comment:
do the tag names really change when interspersed with text/content of a different language!?!? i.e. `{(lien)}` instead of `{(link)}`?

Reply:
LOL

Reply:
the second tag is the text dsiplayed. We wanted the french version to show `lien`